apps/site/src/app/portal/Portal.tsx

```diff
@@ -1,21 +1,50 @@
-import api from "@/lib/utils/api";
+import { redirect } from "next/navigation";
+import type { Metadata } from "next";
 
-async function getIdentity(): Promise<string> {
-	const res = await api.get<string>("/user/me");
-	return res.data;
+import VerticalTimeline from "./VerticalTimeline";
+import Message from "./Message";
+import getUserIdentity from "@/lib/utils/getUserIdentity";
+
+import styles from "./Portal.module.scss";
+
+export const enum PortalStatus {
+	pending = "PENDING_REVIEW",
+	reviewed = "REVIEWED",
+	accepted = "ACCEPTED",
+	rejected = "REJECTED",
+	waitlisted = "WAITLISTED",
+	confirmed = "CONFIRMED",
 }
 
+export const metadata: Metadata = {
+	title: "Portal | IrvineHacks 2024",
+};
```

Comment:
Chore/Issue: the metadata needs to be exported from the `page.tsx` instead of the component. See [`app/login/page.tsx`](https://github.com/HackAtUCI/irvinehacks-site-2024/blob/main/apps/site/src/app/login/page.tsx) for an example.